css/main.css

```diff
@@ -416,6 +421,62 @@ textarea {
     brightness(92%) contrast(82%);
 }
 
+@media (max-width: 720px) {
+  .menu-bars {
+    cursor: pointer;
+    z-index: 200;
+  }
+
+  #desktop-nav {
+    z-index: 100;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+    background: #3c3a39;
+    padding: 3.5rem 2.5rem;
+  }
+
+  #desktop-nav ul li {
+    text-align: left;
+    margin-bottom: 0.6rem;
+  }
+
+  /* #desktop-nav ul li {
+    text-align: right;
+    margin-bottom: 3rem;
+    padding: 1rem;
+  } */
```

Comment:
- [ ] I would suggest removing this commented code also, in order to keep your code clean :eyeglasses: